src/app/tabs/run-tab.js

```diff
@@ -225,13 +225,19 @@ function makeRecorder (appAPI, appEvents, opts, self) {
             modalDialogCustom.alert('Failed to create file ' + newFile)
           } else {
             appAPI.switchFile(newFile)
+            self._view.recorderCount.innerText = 0
```

Comment:
you are putting the record count to 0 but are you sure that the actually records is reset?

Reply:
commit https://github.com/ethereum/remix-ide/pull/1219/commits/41790513f833ebbe7e90f77fc127f2ae9ac77abf